lib/components/CursorTooltip/CursorTooltip.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useRef } from 'react'
+import Tooltip from '../Tooltip'
+import { EMPTY_STRING } from '../../consts'
+
+export type CursorTooltipProps = {
+  children: React.ReactElement
+  clear?: boolean
+  data?: React.ReactElement | string
+  extraClass?: string
+  extraPopperClass?: string
+  enterDelay?: number
+  disappearAfterTimeout?: number
+} & React.ComponentProps<typeof Tooltip>
+
+function CursorTooltip({
+  children,
+  clear = false,
+  data = EMPTY_STRING,
+  extraClass = EMPTY_STRING,
+  extraPopperClass = EMPTY_STRING,
+  enterDelay = 1000,
+  ...rest
```

Comment:
it seems like `clear` can be part of `rest`, since it's not used in `CursorTooltip`.
about `enterDelay` - doesn't the one in the `Tooltip` (400) override this one? If it does, need to pass it to the Tooltip as well

Reply:
I think that since we are using the "`enterDelay`" internally, we don't need the external value, but we can pass it down for readability. About clear, we are not using it here, but I thought to leave the API similar to the regular `tooltip`

Reply:
I mean you still will be able to pass clear to `CursorTooltip`, but it will be a part of `rest`. It will be possible to access it in `Tooltip` this way, but there's no use in `clear` in the `CursorTooltip` as far as I can see

Reply:
ok, done thanks